tests/integration/aps/test_fetch_from_api.py

```diff
@@ -0,0 +1,78 @@
+from datetime import datetime
+
+import pytest
+from airflow import DAG, AirflowException
+from airflow.models import DagBag
+from airflow.models.dagrun import DagRun
+from airflow.utils.state import DagRunState
+from aps.APSApiClient import APSApiClient
+from aps.APSParams import APSParams
+from aps.fetch_from_api import save_file_in_s3
+from aps.repository import APSRepository
+from busypie import SECOND, wait
+
+DAG_NAME = "aps_fetch_api"
+TRIGGERED_DAG_NAME = "aps_process_file"
+
+
+@pytest.fixture
+def dag():
+    dagbag = DagBag(dag_folder="dags/", include_examples=False)
+    assert dagbag.import_errors.get(f"dags/{DAG_NAME}.py") is None
+    return dagbag.get_dag(dag_id=DAG_NAME)
+
+
+def test_dag_loaded(dag: DAG):
+    assert dag is not None
+    assert len(dag.tasks) == 3
+
+
+@pytest.mark.vcr(record_mode="new")
```

Comment:
I don't see any `new` mode in the [docs](https://vcrpy.readthedocs.io/en/latest/usage.html#once), we should use `once` instead.